modules/ref_fb_module/tests/test_ref_fb_module.cpp

```diff
@@ -150,3 +153,11 @@ TEST_F(RefFbModuleTest, StatisticsNumOfSignals)
     auto fb = module.createFunctionBlock("ref_fb_module_statistics", nullptr, "id");
     ASSERT_EQ(fb.getSignalsRecursive().getCount(), 3u);
 }
+
+TEST_F(RefFbModuleTest, CreateFunctionBlockClassifier)
```

Comment:
Missing integration tests that generate standard inputs and validate that the classification outputs are correct for both synchronous and asynchronous signals. Test cases with invalid input signals should also be added.

I'd try to think about when the function block will fail, and add said cases to the test suite as well - marked as "DISABLED" until the functionality to handle them is added to the classifier.